src/test/java/uk/gov/justice/hmpps/prison/service/CaseNoteMovementServiceImplIntTest.java

```diff
@@ -16,6 +17,7 @@
 
 @ActiveProfiles("test")
 @SpringBootTest
+@WithMockAuthUser("")
```

Comment:
this test then started failing as no authentication in the session (invalid with new authentication holder).  Using `""` then ensures that the `username` is not set and `clientId` is set, which is a more appropriate test